src/snovault/elasticsearch/searches/mixins.py

```diff
@@ -109,7 +113,8 @@ def _format_aggregation(self, facet_name):
             TERMS: self._get_aggregation_bucket(facet_name),
             TOTAL: self._get_aggregation_total(facet_name),
             TYPE_KEY: self._get_facet_type(facet_name),
-            APPENDED: JS_FALSE
+            APPENDED: JS_FALSE,
+            OPEN_ON_LOAD: self._get_facet_open_on_load(facet_name),
```

Comment:
Should also update tests for `_format_aggregation` and `_format_aggregations`: 
https://github.com/ENCODE-DCC/snovault/blob/84c439a24587354903fff5977025795198ce1112/src/snovault/tests/test_searches_mixins.py#L1559
https://github.com/ENCODE-DCC/snovault/blob/84c439a24587354903fff5977025795198ce1112/src/snovault/tests/test_searches_mixins.py#L1592

I think you can do this by adding a `open_on_load: true/false` to the facet fixture: https://github.com/ENCODE-DCC/snovault/blob/84c439a24587354903fff5977025795198ce1112/src/snovault/tests/test_searches_mixins.py#L5

And then asserting that expected['open_on_load'] == actual['open_on_load'] in the tests.